perfkitbenchmarker/benchmark_sets.py

```diff
@@ -190,7 +191,7 @@ def GetBenchmarksFromFlags():
     else:
       benchmark_names.add(benchmark)
 
-  # Expand recursive sets
+# Expand recursive sets
```

Comment:
As far as I understand it doesn't use the `flake8` version installed within the `tox` folder. The pre-commit hook executes `check` which then calls `check-lint.sh`. In that script, at first it complains if you don't have `flake8` installed, then it uses the `flake8` version installed on the system (`/usr/bin`), not the one within the `tox` folder. Am I missing something here?